notebooks/log_and_register_custom_model.py

```
@@ -0,0 +1,123 @@
+
+mlflow.set_experiment(experiment_name=model_experiments_name)
+git_sha, current_branch = get_git_info()
+
+with mlflow.start_run(tags={"branch": current_branch, "git_sha": git_sha}) as run:
+    run_id = run.info.run_id
+    signature = infer_signature(model_input=X_train, model_output=example_prediction)
+    table_name = f"{catalog_name}.{schema_name}.train_set"
+    version = get_latest_delta_version(table_name, spark)
+    dataset = mlflow.data.from_spark(train_set_spark, table_name=table_name, version=version)
+    mlflow.log_input(dataset, context="training")
+    dist_path = f"/{config.volumes_root}/{config.catalog_name}/{config.schema_path}/dist/mlops_with_databricks-0.0.1-py3-none-any.whl"
```

Ruff

Line too long (134 > 120)

(E501)